templates/docs/base/code.md

```diff
@@ -38,10 +38,18 @@ View example of the base code block
 
```

Comment:
Just noticed ^ I don't think we use plurals and title case in headings, so probably should be "Code snippet"